components/eamxx/src/physics/shoc/shoc_functions_f90.cpp

```diff
@@ -2856,6 +2861,7 @@ Int shoc_main_f(Int shcol, Int nlev, Int nlevi, Real dtime, Int nadv, Int npbl,
                                              qwthl_sec_d, wthl_sec_d, wqw_sec_d, wtke_sec_d,
                                              uw_sec_d,    vw_sec_d,   w3_d,      wqls_sec_d,
                                              brunt_d,     isotropy_d};
+  SHF::SHOCRuntime shoc_runtime_options;
```

Comment:
From comments above, I think it would be better here to set the default value since this function is only used for BFB tests with F90, so it shouldn't need variable parameters.